cloudbuild_public.yaml

```diff
@@ -0,0 +1,66 @@
+steps:
+- name: 'gcr.io/kaniko-project/executor:v1.5.1'
+  id: runtime-image
+  args:
+    [
+      '--cache=true',
+      '--build-arg',
+      'SOURCE_COMMIT=$COMMIT_SHA',
+      '--destination=us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+      '-f', './Dockerfile',
+      '--target=application'
```

Comment:
I guess this is obsolete now as we use the same image for testing and just install `pytest` at container start?

Reply:
Actually no, this is still needed for the CI github actions

Reply:
I'd really like to remove the multi-stage Docker file, at least for testing image.

Reply:
Ok, I can split it into two files. What's the reasoning for preferring two files too a multi stage Dockerfile?